<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="BtnToGameFlashcard">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>312</width>
     <height>157</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgmenu/btnfresized.png);
font: 700 18pt &quot;Leelawadee UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnToGameChoose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>311</width>
     <height>157</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 18pt &quot;Leelawadee UI&quot;;
background-image: url(:/bgmenu/btnchoresized.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnToGameApestype">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>420</y>
     <width>312</width>
     <height>157</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Courier New&quot;;
background-image: url(:/bgmenu/2btnapes.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgmenu/bgmenuedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>308</width>
     <height>154</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgmenu/btnwindowresized.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>308</width>
     <height>154</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgmenu/btnwindowresized.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>308</width>
     <height>154</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgmenu/btnwindowresized.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>BtnToGameFlashcard</zorder>
  <zorder>BtnToGameChoose</zorder>
  <zorder>BtnToGameApestype</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
